app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +69,121 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("A cell that is dead can be set to alive")
+    void setAliveAt() {
+        Board board = new Board();
+
+        Boolean cellStatus = board.isCellAliveAt(1, 1);
+        assertFalse(cellStatus);
+
+        board.setAliveAt(1, 1);
+        cellStatus = board.isCellAliveAt(1, 1);
+
+        assertTrue(cellStatus);
+
+    }
+
+    @Nested
+    @DisplayName("Tests for getAliveNeighborsCountOf")
+    class GetAliveNeighborsCountOf {
+        public Board board;
+        @BeforeEach
+        void createBoard() {
+            board = new Board(3,3);
+        }
+        @Test
+        @DisplayName("A cell with 0 alive neighbor should return 0 alive neighbor")
+        void getAliveNeighborsCountOf() {
+            int numOfAliveNeighbors = board.getAliveNeighborsCountOf(1,1);
+            assertEquals(numOfAliveNeighbors, 0);
+        }
+
+        @Test
+        @DisplayName("A cell with 1 alive neighbor should return 1 alive neighbor")
+        void getAliveNeighborsCountOfTest1() {
```

Comment:
We should probably use a more descriptive name here. I'm not sure what the standard practice is for this format. This applies to all the test method names.

Reply:
Id have to look into this more cause with the display names it looks like you usually just put the name of the method but we cannot use the same method multiple times. Im sure there's some nested testing feature that I have not discovered yet that does this. Ill add a card for this

Reply:
It looks like I should still follow naming conventions. There are a few but I went with [MethodName_StateUnderTest_ExpectedBehavior] which is shown in my most recent push to this branch